policyengine_us/parameters/gov/states/il/tax/income/exemption/personal.yaml

```diff
@@ -9,5 +9,9 @@ metadata:
   reference:
```

Comment:
Could you add the 2025 value:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/f6c81cc8-f01d-457f-91d6-d9b0c3fc686a)
from the legal code